syntax = "proto3";

package common;

message VectorClock {
  repeated int32 clock = 1;
}
